src/Avalonia.Themes.Fluent/Avalonia.Themes.Fluent.csproj

```diff
@@ -10,6 +10,15 @@
     <AvaloniaResource Include="**/*.xaml" />
     <AvaloniaResource Include="Assets\*" />  
   </ItemGroup>
+  <ItemGroup>
+    <None Remove="Controls\NativeMenuBar.xaml" />
+    <None Remove="NativeMenuBar.xaml" />
+  </ItemGroup>
+  <ItemGroup>
+    <AvaloniaResource Update="Controls\NativeMenuBar.xaml">
+      <Generator>MSBuild:Compile</Generator>
+    </AvaloniaResource>
+  </ItemGroup>
```

Comment:
```suggestion
```

You don't need it. Just a garbage generated by Visual Studio.

Reply:
Sorry: My first ever Pull Request to an open-source projekt. Should have seen this.